html/semantics/scripting-1/the-script-element/resources/slow-flag-setter.py

```diff
@@ -0,0 +1,15 @@
+
+import time
+
+def main(request, response):
+  time.sleep(1)
+
+  headers = [("Content-Type", "text/javascript")]
+
+  if request.GET.first("result", "successful") == "successful":
+    body = "window.didExecute = true;"
+  else:
+    headers.append(("Transfer-encoding", "chunked"))
+    body = "Invalid\n\Chunk\n"
```

Comment:
Is the `\Chunk` intended here? I assume it doesn't affect the results but it looks potentially like a typo.

Reply:
Should be just a typo. Fixed.